index.bs

```diff
@@ -668,7 +668,7 @@ The [=sampling frequency=] differs from the [=requested sampling frequency=] in
    [=device sensor=]'s measurements are skipped and the [=sensor readings=] are not updated.
 
 The <dfn>reporting frequency</dfn> for a concrete {{Sensor}} object is defined as a frequency at which
-{{Sensor/onreading}} notification is called.
+the {{Sensor/onreading}} {{EventHandler}} is called.
```

Comment:
Maybe use [Fire event concept](https://dom.spec.whatwg.org/#concept-event-fire) that should cover dispatching of an event for event listeners / event handler.

Reply:
makes sense, thanks!